### DISCUSSION TITLE
Transactions done between my own savings accounts

### DISCUSSION THREAD

**Discussion by ayushsoni1001** [ananthakumaran/paisa#282] [General]
First of all thanks for making this amazing tool! I was resisting on getting started but I'm glad I did. 

Now can someone help me figure out how should I mark and handle transactions done between two of my own savings accounts?

**Comment by ayushsoni1001** [comment]
Found a way

Instead of Expense: or Income: we can make both line items as Assets:Checking:[Your bank]

**Comment by arpit-saxena** [comment]
I have a similar problem. When I ingest statements of both the accounts, I now end up with 2 entries transferring money from account 1 to account 2, one in account 1's ledger and other in the second one's. Is there any automated way to handle this?
